### PR TITLE
Re-adding the Methanol Byproduct

### LINKED ISSUE #53836: Methanol Byproduct from Crafting Moonshine Mash
### Describe the bug

Crafting Moonshine Mash with:
15 Water
12 Pumpkin
50 Sugar
2 Yeast
Will result in 105 Mash and a byproduct of 68750 Methanol (or 68.75L of Methanol)
The recipe, according to HHG, will make a byproduct of only 275 Methanol. I suspect there's a mistake in the recipe.

### Steps to reproduce

1. Make a world, spawn in 15 units of water in any container, 12 Pumpkin, 50 Sugar, 2 yeast, and the 'family cookbook' book for the recipe.
2. Craft a single batch of moonshine mash.
3. Watch as the game gives you 68L of Methanol.

### Expected behavior

I expected the byproduct to result in a more reasonable amount, like just 2L or something.

### Screenshots

IMG_20211228_184226
Screenshot_2021-12-28-18-54-58-13_a77f7b094295b5a083bf400fa615cbfb

### Versions and configuration

- OS: Android 11, ColorOS 11.1
- Game Version: Stable 0.F-3
- Graphics Version: Tiles
- In-game Language: English
- Mods Loaded: dda, Disable NPC Needs, Bionic Professions, Magiclysm, No Fungal Monsters, Fuji's Military Profession Pack, Magiclysm No Class Limit, Speedydex, Stats Through Kills, Stats Through Skills

### Additional context

I'm all in for mass producing biodiesel but 68L Methanol from really cheap materials is just too much imo.

### PR DESCRIPTION
**Pull request by Somus88**
Re-adding Methanol byproduct, also with the fixed value.

Related, Issue number #53836

#### Summary
Content "Re-adding the Methanol Byproduct from Crafting Moonshine Mash"

#### Purpose of change
To fix the methanol byproduct amount. (creates 68L of Methanol from water, pumpkin, sugar, and yeast.)
Should Fix #53836 
Steps to Reproduce:

1. Make a world, spawn in 15 units of water in any container, 12 Pumpkin, 50 Sugar, 2 yeast, and the 'family cookbook' book for the recipe.
2. Craft a single batch of moonshine mash.
3. Watch as the game gives you 68L of Methanol.

#### Describe the solution
The solution would be to re-add the byproduct and fix the amount produced to 2 units of methanol, which will result in 500mL of methanol, which is around 13-14% of the resulting product.

#### Describe alternatives you've considered
Removal of methanol byproduct entirely in the books however I personally would still like cheap quick methanol :)

#### Testing

None because I don't know how to + it's pretty trivial

### CODE REVIEW

**Review by Maleclypse — APPROVED** [review]

**Comment by Alex-Folts** [comment]
I see 2 problems with this:

- it is unfermented stuff, basically it is just all ingredients mixed together - at this point there is no ethanol and methanol in it, it need to stay for some time to generate it(fermentation need to occur). Means you cant get out methanol at this point.
- 500ml of methanol is too much for about 4L of initial stuff, there is methanol and aceton and some other volatiles, but not of this quantity. Those volatiles people call heads, they come first when you distilling and there not much of them, need to google some numbers but should be much less than this. But again, it will matter on later stage when you distilling **fermented** stuff.

**Comment by Somus88** [comment]
> I see 2 problems with this:
> 
> * it is unfermented stuff, basically it is just all ingredients mixed together - at this point there is no ethanol and methanol in it, it need to stay for some time to generate it(fermentation need to occur). Means you cant get out methanol at this point.
> * 500ml of methanol is too much for about 4L of initial stuff, there is methanol and aceton and some other volatiles, but not of this quantity. Those volatiles people call heads, they come first when you distilling and there not much of them, need to google some numbers but should be much less than this. But again, it will matter on later stage when you distilling **fermented** stuff.

I have changed where Methanol is produced to the distillation process, and fixed the amount accordingly to Wikipedia (it says 50-150mL of foreshot that contains most of methanol,) unfortunately most other websites I found are rather questionable in terms of reliability.

**Review by Maleclypse — APPROVED** [review]

File: data/json/recipes/food/distill.json [hunk 1]
```diff
@@ -81,8 +81,7 @@
     "type": "recipe",
     "activity_level": "NO_EXERCISE",
     "result": "moonshine",
-    "//": "should produce 12-15% foreshot with significant methanol",
-    "//2": "1 unit of methanol in the JSON is 250 units in game",
+    "byproducts": [ [ "chem_methanol", 1 ] ],
```
Comment (Maleclypse) [inline data/json/recipes/food/distill.json:84]:
```suggestion
    "byproducts": [ [ "chem_methanol", 1 ] ],
    "//": "kick test",
```

File: data/json/recipes/food/distill.json [hunk 2]
```diff
@@ -81,8 +81,8 @@
...
+    "//": "kick test",
```
Comment (Maleclypse) [inline data/json/recipes/food/distill.json:85]:
```suggestion
```

**Pull request merged**